po/bn_IN.po

```diff
@@ -218,49 +205,39 @@ msgstr "ছোট হাতের হরফ তালিকা"
 msgid "Upper Case List"
 msgstr "বড় হাতের হরফ তালিকা"
 
-#: widgets.py:119
-msgid "Rich Text (RTF)"
-msgstr "রিচ টেক্সট (RTF)"
-
-#: widgets.py:121
-msgid "RTF"
-msgstr "আরটিএফ"
-
-#: widgets.py:125
-msgid "Hypertext (HTML)"
-msgstr "হাইপারটেক্সট (এইচটিএমএল)"
+#: fontcombobox.py:90
+msgid "Select font"
+msgstr ""
 
-#: widgets.py:127
-msgid "HTML"
-msgstr "এইচটিএমএল"
+#: AbiWordActivity.py:69
+msgid "Connecting..."
+msgstr ""
 
-#: widgets.py:132
-msgid "Plain Text (TXT)"
-msgstr "প্লেইন টেক্সট (TXT)"
+#: AbiWordActivity.py:99
+msgid "Edit"
+msgstr "সম্পাদন"
 
-#: widgets.py:134
-msgid "TXT"
-msgstr "টিএক্সটি"
+#: AbiWordActivity.py:105
+msgid "View"
+msgstr "প্রদর্শন"
 
-#: widgets.py:138
-msgid "Portable Document Format (PDF)"
-msgstr ""
+#: AbiWordActivity.py:120
+msgid "Text"
+msgstr "টেক্সট"
 
-#: widgets.py:140
-msgid "PDF"
+#: AbiWordActivity.py:126
+msgid "Paragraph"
 msgstr ""
 
-#~ msgid "Cancel"
-#~ msgstr "বাতিল"
-
-#~ msgid "Image"
-#~ msgstr "ছবি"
-
-#~ msgid "Format"
-#~ msgstr "ফর্ম্যাট"
+#: AbiWordActivity.py:132
+msgid "Table"
+msgstr "টেবিল"
 
-#~ msgid "Choose image"
-#~ msgstr "ছবি বেছে নিন"
+#: AbiWordActivity.py:136
+msgid "Insert Image"
+msgstr "ছবি ঢোকান"
 
-#~ msgid "Style: "
+#: AbiWordActivity.py:143
+msgid "Floating"
+msgstr ""
 #~ msgstr "স্টাইল:"
```

Comment:
@Himanshu40, removing Write from `sugar-build` is not the right way to solve the problem.  In future, please find the real problem and work it.  :grin:  If it had been solved, then @samswag would not have hit it.